alerts/report_sources/goma/alerts_goma_2019-10-26.Rmd

```diff
@@ -0,0 +1,2469 @@
+## Add population data
+
+Load the population data
+
+```{r load_pop_data}
+
+## load the data
+current_pop 
+pop_data <- rio::import(current_pop)
+glimpse(pop_data)
+
+pop_data <- pop_data %>%
+  clean_data(guess_dates = FALSE) %>%
+  as_tibble()
+```
+
+### Check for area not in alerts
+
+```{r filter_by_alerts}
+
+## load the data
+pop_data <- pop_data %>% 
+  filter(aire_de_sante %in% x$aire_de_sante & zone_de_sante %in% x$zone_de_sante)
+
+pop_data
+
+## aire de sante population
+aire_de_sante_pop <- pop_data %>%
+  mutate(as_population = population) %>%
+  select(zone_de_sante, aire_de_sante, as_population)
+
+## zone de sante population
+zone_de_sante_pop <- pop_data %>%
+  group_by(zone_de_sante) %>%
+  summarise(zs_population = sum(population, na.rm = TRUE))
+
+total_population <- sum(unique(pop_data$population), na.rm = TRUE)
+
+## adding populations to main data
+x <- x %>%
+  left_join(aire_de_sante_pop, 
+            by = c("aire_de_sante", "zone_de_sante")) %>%
+  left_join(zone_de_sante_pop, 
+            by = c("zone_de_sante")) 
+  
+if(!allNA(x$as_population)){
+  x <-   x %>% 
+    mutate(total_population = total_population)
+}
+
+
+```
+
+
+<!-- ======================================================= -->
+## Check population linkage (review regularly)
+
+```{r check_population_linkage}
+
+# Calculating number of individuals successfully linked to population data
+number_individuals <- nrow(x)
+number_individuals_aire_linked <- sum(!is.na(x$as_population))
+prop_linked <- sum(number_individuals_aire_linked / number_individuals)
+
+# Compare these two quantities - ensure there aren't any mispellings we're missing
+table(x$aire_de_sante[!number_individuals]) # aire de santes in database we're not matching
+
+# Identifying Aire de Santes with missing population data
+goma_ha_pop <- pop_data$population
+index <- which(is.na(goma_ha_pop))
+non_pop_data <- pop_data$aire_de_sante[index]
+
+# Identifying number of alerts linked to Aire de Sante with 
+# missing population data
+alerts_no_pop <- sum(x$aire_de_sante %in% non_pop_data)
+
+```
+
+For the Goma Alerts Database, **`r round(100 * prop_linked, 2)`%** of individuals 
+(`r number_individuals_aire_linked` / `r number_individuals`) had a legitimate Aire 
+de Sante belonging to Goma inputted - the remainder had Aire de Santes which either
+do not belong to Goma or are not an Aire de Sante. **`date_na`** alerts were removed
+for an invalid date.
+<!-- REVIEW: invalid dates removed ^ -->
+
+**Note:** Population data is unavailable for the aire de santes of 
+**`r non_pop_data`** which are not present in the population database. These
+Aire de Santes make up **`r alerts_no_pop`** alerts. 
+
+**Note:** These analyses **CANNOT** detect legitimate Aire de Santes missing 
+from the geographical/population database. If any of these are identified as
+missing, they must be manually added to the database so that they can be
+incorporated into the analyses presented here. 
```

Comment:
Streamline this, we don't need that much detail, plus it's confusing anyway. Just say that for alert incidences, we need pop data and known health zones / areas. Other stuff is ignored in these analyses, and this is how many alerts are thus lost.